pages/static/src/js/app.js

```diff
@@ -1,5 +1,4 @@
 window.$ = window.jQuery = require('jquery');
-window.ListFuzzySearch = require('./vendor/list.fuzzysearch.min.js');
```

Comment:
remove file too?

Reply:
Didn't want to due to caching reasons, if a user has old page and it tries to access vendor folder we'll error out in the off chance that's not cached.

Reply:
if a user has old page then don't they also have the old js files?